cmd/tuf/app/init.go

```diff
@@ -240,6 +240,11 @@ func InitCmd(ctx context.Context, directory, previous string,
 		}
 	}
 
+	// Reset and delegations: they will be updated in DelegationCmd.
+	if err := repo.ResetTargetsDelegations("targets"); err != nil {
+		return err
+	}
+
```

Comment:
I was expecting this PR to be all orchestration changes so was surprised to see this change to the app code. Is this just to make super certain there aren't any delegations?

Reply:
Yes, this is because otherwise, we maintain the previous delegations in the targets file (they carry forward from any previous targets.json)

Reply:
Is there an easy way to add test(s) to ensure we don't see any delegations in the output?

Reply:
done!! thank you!